api.md

```diff
@@ -840,7 +846,150 @@ SELECT reorder_chunk('_timescaledb_internal._hyper_1_10_chunk', 'conditions_devi
 runs a reorder on the `_timescaledb_internal._hyper_1_10_chunk` chunk using the `conditions_device_id_time_idx` index.
 
 ---
+## Continuous Aggregates :community_function: [](continuous-aggregates)
+TimescaleDB allows users the ability to automatically recompute aggregates 
+at predefined intervals and materialize the results. This is suitable for 
+frequently used queries.
+
+> - [CREATE VIEW](#cagg_create_view)
```

Comment:
Adding the  `>` puts these into a strange format.  This should be a normal list.

![Screen Shot 2019-05-02 at 5 30 42 PM](https://user-images.githubusercontent.com/12288273/57108246-1ddd9800-6d00-11e9-8f93-310766352584.png)

Reply:
modified.